docs/source/format/driver_manifests.rst

```diff
@@ -19,47 +19,59 @@
 ADBC Driver Manager and Manifests
 =================================
 
-.. note:: This document focuses on scenarios that use the driver manager
-          to load drivers.  The driver manager is not required to use ADBC
-          in general, but allows a convenient experience for dynamically
-          loading arbitrary drivers.
+.. note:: This document describes using the :term:`driver manager` to load
+          drivers.  The driver manager is not required to use ADBC in general
+          but it allows loading drivers written in a different language from the
+          application and improves the experience when using multiple drivers in
+          a single application. For more information on how the driver manager
+          works see :doc:`how_manager`.
 
-The ADBC driver manager is itself, an ADBC driver which loads another driver
-dynamically and forwards the calls to the loaded driver.  For more information on the
-driver manager see :doc:`how_manager`.
-
-There are two ways to specify a driver for the driver manager to load:
+There are two ways to load a driver with the driver manager:
 
 1. Directly specifying the dynamic library to load
 2. Referring to a driver manifest file which contains metadata along with the
    location of the dynamic library to be loaded
 
-When using the driver manager, you can either use the ``driver`` option to the
-driver manager, or you can use functions in the language bindings which explicitly
-load a driver by name.
+With either method, you specify the dynamic library or driver manifest as the
+``driver`` option to the driver manager or you can use an explicit function for
+loading drivers if your driver manager library exposes one (e.g., C++, see
+example below).
 
-.. note:: In addition to the ``driver`` option, there is also an ``entrypoint`` option
-          which can be used to specify the entrypoint function to call for populating
-          the driver function table.  If the driver does not use the default entrypoint
-          function, it can be indicated with this option.
+.. note:: In addition to the ``driver`` option, there is also an ``entrypoint``
+          option that should be used if the driver uses a non-default
+          entrypoint.
```

Comment:
Should "entrypoint" get its own glossary term?

Reply:
I suppose it could, so we can be clear about what we mean by it.

Reply:
Done in e3c517554. How's that?

Reply:
I changed it from func -> type.